cargo-workspaces/src/utils/pkg.rs

```diff
@@ -86,15 +86,19 @@ impl Listable for Vec<Pkg> {
     }
 }
 
-pub fn get_pkgs(metadata: &Metadata, all: bool) -> Result<Vec<Pkg>> {
+pub fn get_pkgs(metadata: &Metadata, all: bool, exclude_lib: bool) -> Result<Vec<Pkg>> {
```

Comment:
not a huge fan of bools as function parameters - especially as it grows and we see `false` being passed in rename.rs and changable.rs

would the maintainers prefer a struct config with some defaults instead?